source/optional_features/digital_wallets/index.html.md.erb

```diff
@@ -0,0 +1,70 @@
+
+If you use Stripe, [ePDQ](/switching_to_live/set_up_a_live_epdq_account/#set-up-a-live-epdq-account) or [Smartpay](/switching_to_live/set_up_a_live_smartpay_account/#set-up-a-live-smartpay-account) as your PSP, you cannot enable Apple Pay or Google Pay.
+
+## Enable Apple Pay
+
+Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
+
+1. Go to __Settings__ and select __Card types__.
+
+1. Select the __Digital wallet__ tab.
+
+1. Select __Enable__ next to Apple Pay.
+
+1. On the __Enable Apple Pay__ screen, select __Yes, turn on Apple Pay__.
+
+## Enable Google Pay
+
+To enable Google Pay, you must enter a unique Google Pay merchant ID into your GOV.UK Pay account.
+
+You get your Google Pay merchant ID from your Worldpay merchant admin interface.
+
+1. If required, speak to your Worldpay account manager to upgrade your Worldpay merchant admin interface, so that the interface has Google Pay functionality.
+
+1. Sign in to your [Worldpay merchant admin interface](https://secure.worldpay.com/sso/public/auth/login.html) [external link].
+
+1. Go to __Integration__ and select the __Pay with Google__ tab.
+
+1. Under __Generate unique merchant ID__, select __Generate__.
+
+1. Copy the generated merchant ID.
+
+Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
+
+1. Go to __Settings__ and select __Card types__.
+
+1. Select the __Digital wallet__ tab.
+
+1. Select __Enable__ next to Google Pay.
+
+1. On the __Enable Google Pay__ screen, select __Yes, turn on Google Pay__.
+
+1. Enter the generated merchant ID and select __Submit__.
+
+## 3D Secure
+
+If a digital wallet transaction requires 3D Secure authentication, your users must complete 3D Secure authentication in the same way as non-digital wallet transactions.
+
+## Restrictions
+
+The GOV.UK Pay platform cannot apply [corporate card surcharges](/optional_features/corporate_card_surcharges/#corporate_card_surcharges) to digital wallet transactions made with a corporate card.
+
+When you look at individual digital wallet transactions in your GOV.UK Pay account transactions list, you will only see the last 4 digits of the card number.
+
+The expiry date for individual digital wallet transactions in your GOV.UK Pay account transactions list is the expiry date of the XYZ certificate rather than the payment expiry date.
+
+### Google Pay
+
+You cannot restrict Google Pay digital wallet transactions by card type. For example, you cannot only accept digital wallet transactions made with a debit card.
+
+When you look at individual digital wallet transactions in your GOV.UK Pay account transactions list, you will only see the cardholder name if you have enabled [billing address collection](/integrate_with_govuk_pay/#collecting-your-users-39-billing-addresses).
```

Comment:
This isn't true. We collect the name and email address of the user when the user makes a card payment. So the name will appear on the transactions list even if the billing address wasn't collected.

Reply:
Although, strictly speaking, the user can enter a dummy name, so it's not necessarily the real cardholder name.

Reply:
Just to build consensus, is everyone happy to remove this? @katiebates @alexbishop1 ?

Reply:
It depends on how likely it is that the name will be different to the cardholder name and whether this matters. Where does the name come from? Is it the name associated with the Google account?

I can think of a few reasons why the cardholder name might differ from the Google account name:
- Minor differences like the cardholder name being “Mr Alexander Bishop” and the Google account name being “Alex Bishop”
- The Google account name being completely different to the cardholder name e.g. an alias — I don’t know how hard Google tries to persuade people to provide their real name
- The Google account being in the name of one person but the cardholder name being in the name of another person — I can imagine this might be common for families (Google account in the name of one family member, card in the name of another)

The most important question is whether services will actually care about any differences?

Reply:
I have no idea. I think that is a product question. From a content perspective, I don't think it makes a difference whether it is included or not.

Reply:
Removing as per discussion with @katiebates